code/web/cypress/integration/Subscription_spec.js

```diff
@@ -0,0 +1,37 @@
+//baseURL represents the landing page of "Profile" which displays "My Account"
+const baseURL = 'http://localhost:3000/crates';
+const subscriptions = 'http://localhost:3000/user/subscriptions';
+
+describe('Subscriptions', () => {
+  beforeEach(() => {
+    cy.visit(baseURL);
+    //Login to Seeder
+    //Seeder contains subscriptions
+    cy.get('header div a').contains('Login').click()
+    cy.get('div div')
+      .get('input[type=email]')
+        .type('user@crate.com')
+      .get('input[type=password]')
+        .type('123456')
+      .get('button').contains('Login').click()
+  })
+
+  it('Should be able to subscribe to a new crate', () => {
+    cy.get('.jsx-511674265').first()
+      .get('button').first().click()
+      .location().should(loc => {
```

Comment:
I haven't used this one before! Cool.

Reply:
The .location is super helpful! It's a nice confirmation that the url and the ux match.